apps/notifications/package.json

```diff
@@ -29,12 +29,15 @@
 		"@automattic/calypso-color-schemes": "workspace:^",
 		"@automattic/calypso-polyfills": "workspace:^",
 		"@automattic/webpack-extensive-lodash-replacement-plugin": "workspace:^",
+		"@babel/helpers": "^7.16.3",
```

Comment:
This should be a `devDependency`. We may need to add all the extra `@babel/*` packages referenced in the babel config file.

Reply:
D'oh, you're absolutely right, this is dev time! 🤦 

I've moved it to `devDependencies`, and I've added the other packages as well 👍